src/App.js

```diff
@@ -2,16 +2,18 @@ import React from 'react';
 import Calculator from './components/Calculator';
 import './App.css';
 
+const initAttrObj = {
+  total: null,
+  next: null,
+  operation: null,
+};
+
 // eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] <b>REQUIRED:</b> You overused `eslint-disable`. it's like making changes in Eslint config. they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/) and [Zendesk](https://microverse.zendesk.com/hc/en-us/articles/1500002085202-February-2nd-2021-linter-errors-as-a-reason-of-project-invalidation-)). kindy, remove that comment, it is even useless because the component you are using is already stateless-function.